server/lib/allowed-features.ts

```diff
@@ -5,7 +5,6 @@ import FEATURE from '../constants/feature';
 
 // Please refer to and update https://docs.google.com/spreadsheets/d/15ppKaZJCXBjvY7-AjjCj3w5D-4ebLQdEowynJksgDXE/edit#gid=0
 const FeatureAllowedForTypes = {
-  // Collective page features
```

Comment:
@sbinlondon removed this comment because these features are not only for the collective page (updates for example has standalone pages). Let me know if you still think we want to make this distinction